src/php/Reader.php

```diff
@@ -144,7 +144,7 @@ public function readExpression(Generator $tokenStream) {
                     $this->fnArgs = [];
                     $body = $this->readList($tokenStream, Token::T_CLOSE_PARENTHESIS);
 
-                    if (count($this->fnArgs) > 0) {
+                    if (!empty($this->fnArgs)) {
```

Comment:
This was the unique error that psalm complained once I changed to `errorLevel=4`.